packages/core/src/styles/layout/shims/_gap.scss

```diff
@@ -27,14 +27,14 @@
   @each $size, $sizeValue in $cds-layout-sizes {
     [#{$layout}~='#{$vertical}'][#{$layout}*='#{$gap}#{$breakpoint}:#{$size}'] {
       & > *,
-      &::slotted(*) {
-        margin-top: $sizeValue !important;
+      & > ::slotted(*) {
+        margin-top: $sizeValue;
       }
 
       & > *:first-child,
-      &::slotted(*:first-child),
-      &::slotted([slot]:first-of-type) {
-        margin-top: 0 !important;
+      & > ::slotted(*:first-child),
+      & > ::slotted([slot]:first-of-type) {
+        margin-top: 0;
```

Comment:
The specificity had to be lowered for Safari with the gap shim. This is so the `align:*` utils can override the margin.